### DISCUSSION TITLE
Cannot get app to connect to Immich on Umbrel

### DISCUSSION THREAD

**Discussion by sovereignalmida** [immich-app/immich#5588] [Q&A]
Umbrel app version 1.88.2  IOS App version 1.90.0  I CAN connect vie the browser using my tailscale address and port number but cannot connect via the app.  Is this expected?  Is there a work around?

**Comment by bo0tzz** [comment]
Please make sure the app and server are on the same version and then try again.

**Comment by sovereignalmida** [comment]
Not possible - Umbrel apps cannot be individually updated by the end user. only via the official app store submitter 😜

**Reply by bo0tzz** [reply]
Then you'll need to wait for the latest version to be available, we can't provide support for mismatched app/server versions as that's often the reason things don't work.

**Comment by sovereignalmida** [comment]
Ok - I accept thats the way it is however I would consider that a bug. Backwards compatibility should be expected.

Ill reach out to the app store submitter and see about updating to the latest version of the server.

**Reply by bo0tzz** [reply]
From the top of the github readme:

> ⚠️ The project is under very active development.
> ⚠️ Expect bugs and breaking changes.
> ⚠️ Do not use the app as the only way to store your photos and videos.
> ⚠️ Always follow [3-2-1](https://www.backblaze.com/blog/the-3-2-1-backup-strategy/) backup plan for your precious photos and videos!

Until we decide on a stable release, we're going to be making changes that are not backwards compatible.
